File: Models/Order.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BreakfastInBed.Models
{
    public class Order
    {
        public int CustomerID { get; set; }
        public int RoomNumber { get; set; }
        public int NoOfOccupants { get; set; }
        public IEnumerable<int> BeverageIds { get; set; }
        public IEnumerable<int> PlatterIds { get; set; }
        public string DeliveryStart { get; set; }
        public string DeliveryEnd { get; set; }
    }
}

File: ClientApp/src/app/models/Order.ts
class Order 
{
  constructor() {
    this.customerID = 0;
    this.roomNumber = 0;
    this.noOfOccupants = 0;
    this.beverageIds = [];
    this.platterIds = []
    this.deliveryStart = ""
    this.deliveryEnd = "";
  }
    customerID: number;
    roomNumber: number;
    noOfOccupants: number;
    beverageIds: number[];
    platterIds: number[];
    deliveryStart: string;
    deliveryEnd: string;
};
